src/EVM/Expr.hs

```diff
@@ -950,8 +960,10 @@ eqByte (LitByte x) (LitByte y) = Lit $ if x == y then 1 else 0
 eqByte x y = EqByte x y
 
 min :: Expr EWord -> Expr EWord -> Expr EWord
-min (Lit x) (Lit y) = if x < y then Lit x else Lit y
-min x y = Min x y
+min x y = op2 Min Prelude.min x y
```

Comment:
I think we can use `normArgs` here to make the matches in `simplify` easier to handle?

Reply:
Done!